infra/build/functions/target_experiment.py

```diff
@@ -213,17 +244,25 @@ def main():
   parser.add_argument('--upload_corpus',
                       required=True,
                       help='GCS location to upload corpus.')
+  parser.add_argument('--upload_reproducer',
+                      required=True,
+                      help='GCS location to upload reproducer.')
   parser.add_argument('--upload_coverage',
                       required=True,
                       help='GCS location to upload coverage data.')
   parser.add_argument('--experiment_name',
                       required=True,
                       help='Experiment name.')
+  parser.add_argument('--local_artifact_path',
```

Comment:
Does this need to be a flag at all? When would we need to customize this behaviour?

Reply:
Ah, I was worrying that hardcoding this would affect other usages of this script.
If we don't have to customize it, then I can easily hardcode it.

Reply:
Fixed